app/controllers/communities_controller.rb

```diff
@@ -1,4 +1,5 @@
 class CommunitiesController < ApplicationController
+  layout 'community'
```

Comment:
if you name the layout file `communities.html.haml` it will automatically be picked up for use by this controller.  It will still need to be specified for the indices controller though.